### PR TITLE
[prometheus-kafka-exporter] Add LoadBalancerIp

### PR DESCRIPTION
**Pull request by KimJoonSeo**
#### What this PR does / why we need it
I added an loadBalancerIP configuration to specify a static IP when using NodePort service type.
#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

- This PR has a similar context with PR #3828 .

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by zeritti** [comment]
> I added an loadBalancerIP configuration to specify a static IP when using NodePort service type.

You probably meant _when using LoadBalancer service type_. Using `spec.type=NodePort` with `spec.loadbalancerIP` won't work as the current code does not allow it. Furthemore, the latter applies only to the LoadBalancer type.

File: charts/prometheus-kafka-exporter/values.yaml [hunk 1]
```diff
@@ -71,9 +71,10 @@ service:
   port: 9308
   labels: {}
   annotations: {}
-  # Specifies a custom nodePort for external use.
-  # Must be used with service.type=NodePort.
+  # Specifies a custom nodePort for external use. It must be used with service.type=NodePort.
   nodePort: null
+  # Specifies a reserved static IP. It must be used with service.type=LoadBalancer.
```
Comment (zeritti) [inline charts/prometheus-kafka-exporter/values.yaml:76]:
```suggestion
  # Specifies a reserved static IP. Used with service.type=LoadBalancer.
  # The field is deprecated, implementation-specific annotations should be set instead.
```

File: charts/prometheus-kafka-exporter/values.yaml [hunk 2]
```diff
@@ -71,9 +71,10 @@ service:
   port: 9308
   labels: {}
   annotations: {}
-  # Specifies a custom nodePort for external use.
-  # Must be used with service.type=NodePort.
+  # Specifies a custom nodePort for external use. It must be used with service.type=NodePort.
```
Comment (zeritti) [inline charts/prometheus-kafka-exporter/values.yaml:74]: The field need not be used as it is optional.

```suggestion
  # Specifies a custom nodePort for external use. Used with service.type=NodePort.
```

**Review by zeritti — CHANGES_REQUESTED** [review]

**Comment by KimJoonSeo** [comment]
> > I added an loadBalancerIP configuration to specify a static IP when using NodePort service type.
> 
> You probably meant _when using LoadBalancer service type_. Using `spec.type=NodePort` with `spec.loadbalancerIP` won't work as the current code does not allow it. Furthermore, the latter applies only to the LoadBalancer type.

That's correct. I meant to do it when using LoadBalancer service type.

**Pull request merged**